installfest/general-elixirbridge-info.markdown

```diff
@@ -0,0 +1,25 @@
+### Credits
+
+The ElixirBridge curriculum would not have been possible without the amazing folks who contributed to the [RailsBridge Installfest](http://docs.railsbridge.org/installfest/) and [Intro to Rails Curriculum](http://docs.railsbridge.org/intro-to-rails/).
+The ElixirBridge curriculum is heavily based on the [Intro to Rails Curriculum](http://docs.railsbridge.org/intro-to-rails/), minus the the parts of the curriculum that are Elixir and Phoenix specific. So we wanted to make sure to give credit where credit is due. Thank you to all the people who contributed to the RailsBrige curriculum and general documentation. This has allowed ElixirBridge to get up and running very quickly. For those interested in learning more about RailsBridge checkout their [site](www.railsbridge.org). 
+
+
+
+
+### Elixirbridge Docs
+
+
+### Setup
+
+[Installfest](installfest/01-begin-installfest.html)
```

Comment:
I'm guessing these links will break since they're relative. Whack everything up to and including the slash and they should work fine.